memory/memory_darwin.go

```diff
@@ -22,3 +23,26 @@ func getMemoryInfo() (memoryInfo map[string]string, err error) {
 
 	return
 }
+
+func getMemoryInfoByte() (uint64, uint64, error) {
+	memInfo, err := getMemoryInfo()
+	var mem, swap uint64
+
+	// mem is already in bytes but `swap_total` use the format "5120,00M"
+	if v, ok := memInfo["swap_total"]; ok {
+		idx := strings.IndexAny(v, ",.") // depending on the local either a comma or dot is used
```

Comment:
```suggestion
		idx := strings.IndexAny(v, ",.") // depending on the locale either a comma or dot is used
```

Reply:
This seems fairly fragile given that it depends on locale-specific handling of things. Can we tighten the regex or check that the suffix is `M`? 

PS : Even though it's a fairly unlikely edge case, I do wonder what happens to these values if the locale is RTL.